src/store/contacts/contacts-reducers.ts

```diff
@@ -0,0 +1,32 @@
+import { createReducer } from '@reduxjs/toolkit';
+
+import {
+  addBlacklistedContactAction,
+  addContactAction,
+  addContactRequestAction,
+  deleteContactAction,
+  editContactAction
+} from './contacts-actions';
+import { contactsInitialState, ContactsState } from './contacts-state';
+
+export const contactsReducers = createReducer<ContactsState>(contactsInitialState, builder => {
```

Comment:
we are using an immutable state change approach instead of it's mutation.  
It's bad to combine those 2 approaches at the same time (doc says: [` Immer expects that you will either mutate the existing state, or construct a new state value yourself and return it, but not both in the same function! `)](https://redux-toolkit.js.org/usage/immer-reducers#mutating-and-returning-state).  
  
So, it's better to use single approach right now